### DISCUSSION TITLE
08/05 Wednesday metting zoom link

### DISCUSSION THREAD

**Discussion by ZIYANGSONG2003** [ZIYANGSONG2003/AI-Enhanced-WordPress-Development-Toolkit#93] [Announcements]
Stephanie is inviting you to a scheduled Zoom meeting.

Topic: Stephanie's Zoom Meeting
Time: May 8, 2024 05:00 PM Canberra, Melbourne, Sydney

Join Zoom Meeting
https://anu.zoom.us/j/87063877317?pwd=eU9najIzTkZERkh0TFkrTzFJYjRSZz09

Meeting ID: 870 6387 7317
Password: 340381
One tap mobile
+17193594580,,87063877317#,,,,0#,,340381# US
+12532050468,,87063877317#,,,,0#,,340381# US

Dial by your location
        +1 719 359 4580 US
        +1 253 205 0468 US
        +1 253 215 8782 US (Tacoma)
        +1 301 715 8592 US (Washington DC)
        +1 305 224 1968 US
        +1 309 205 3325 US
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 360 209 5623 US
        +1 386 347 5053 US
        +1 507 473 4847 US
        +1 564 217 2000 US
        +1 646 876 9923 US (New York)
        +1 646 931 3860 US
        +1 669 444 9171 US
        +1 669 900 6833 US (San Jose)
        +1 689 278 1000 US
Meeting ID: 870 6387 7317
Password: 340381
Find your local number: https://anu.zoom.us/u/kbbeB6QopV

Or an H.323/SIP room system:
    Dial: 87063877317@global.zoomcrc.com
    Meeting ID: 87063877317
    H323/SIP Password: 340381

**Comment by zubinlearnspython** [comment]
Thank you for the zoom link
